src/Navigation/Plugins/recastJSPlugin.ts

```diff
@@ -44,7 +44,7 @@ export class RecastJSPlugin implements INavigationEnginePlugin {
      */
     public constructor(recastInjection: any = Recast) {
         if (typeof recastInjection === "function") {
-            recastInjection(this.bjsRECAST);

```

Comment:
is `await Recast()` generate anything in the global namespace? no way to inject an import anymore?

Reply:
I'm not the best to answer these questions :) I build recastjs the same way as ammojs and load it the same way as well.

Reply:
Oh, just saw the ammo init, which has the same log. I assume that you can await this function instead, but i'm not not entirely sure without checking.